broker-cli/commands/orderbook.js

```diff
@@ -58,10 +150,73 @@ function createUI (market, asks, bids) {
       'right': '',
       'right-mid': '',
       'middle': ''
+    },
+    colWidths: [COLUMN_WIDTHS, COLUMN_WIDTHS],
+    style: {
+      'padding-left': 0,
+      'padding-right': 0
+    }
+  })
+
+  gapTable.push([' ', ' '])
+
+  const displayAsks = (asks) => {
+    // Asks are added to the table with highest (worst) price at top, so need to sort
+    [...asks].sort((a, b) => {
```

Comment:
You can just do `asks.slice().reverse()` here instead. Might be a little simpler and probably faster.